Phonebook.Frontend/src/index.html

```diff
@@ -47,5 +46,39 @@
     } else {
       top.location = self.location;
     }
+    msieversion();
+
+    function openURL(e) {
+      browser = e.getAttribute('value');
+      if (browser == 'microsoft-edge') {
+        //edge can not be opened via shell
+        window.open((browser = e.getAttribute('value') + ':' + window.location.href));
+      } else {
+        new ActiveXObject('WScript.Shell').run(browser + ' ' + window.location.href);
+      }
+    }
+
+    function msieversion() {
+      var browserList = {
+        //list of browser-links where app can be openend in: key=browser name, value=shell-command to open browser
+        'Microsoft Edge': 'microsoft-edge',
+        'Google Chrome': 'chrome',
+        'Mozilla Firefox': 'firefox',
+        Opera: 'opera'
+      };
+      var ua = window.navigator.userAgent;
+      var msie = ua.indexOf('MSIE ');
+
+      if (msie > 0 || !!navigator.userAgent.match(/Trident.*rv\:11\./)) {
+        // stone age browser IE
+        var html =
+          '<div class="pb-no-ie"><p><strong>This application does not support Internet Explorer!</strong></p><p>To run the Phonebook, please use one of the following modern browsers:</p><ul>';
+        for (var key in browserList) {
+          html += '<li onclick="openURL(this)" value="' + browserList[key] + '"><strong>' + key + '</strong></li>';

```

Comment:
Wie wärs wenn du hier einfach normale `<a>` tags nimmst? :D
Dann ist es auch barrierefrei.

Reply:
`<a href="https://www.w3schools.com" target="_blank">` ganz ohne Javascript ;)

Reply:
I tried, but there was a reason for taking blank li elements, but I dont remember why :D

Reply:
Yep, but we don't want to open the app in a different browser. Just supply links where they can download them. ActiveX is should not be used anymore.